setup.py

```diff
@@ -1,43 +1,3 @@
-from setuptools import setup, find_packages
+from setuptools import setup
 
-
-def scm_version():
-    def local_scheme(version):
-        if version.tag and not version.distance:
-            return version.format_with("")
-        else:
-            return version.format_choice("+{node}", "+{node}.dirty")
-    return {
-        "relative_to": __file__,
-        "version_scheme": "guess-next-dev",
-        "local_scheme": local_scheme
-    }
-
-
-setup(
-    name="nmigen",
-    use_scm_version=scm_version(),
```

Comment:
use_scm_version and setup_requires=["setuptools_scm"] kwargs "are considered legacy behavior"